[xalan-c] [x64-linux] undefined reference to `xalanc_1_11::XalanMsgContainer::getMessageCount()'

I've built `xalan-c:x64-linux`, but linking it fails with:
```
undefined reference to `xalanc_1_11::XalanMsgContainer::getMessageCount()'
undefined reference to `xalanc_1_11::XalanMsgContainer::getMessage(unsigned long)'
```
because `libxalanMsg.a` isn't present on linker's cmdline. I refer to `xalan-c` like this:
```
find_package(XalanC REQUIRED)
target_link_libraries(<my-target> PUBLIC XalanC::XalanC)
```

from what I can tell cmake files in `~/vcpkg/installed/x64-linux/share/xalan-c` are simply ignored and instead `/usr/local/share/cmake-3.14/Modules/FindXalanC.cmake` is used to find `libxalan-c` and because (long ago) it was written "xalan-c is a shared library" mindset -- it doesn't even try to locate `libxalanMsg`. 

What is the correct way to address this problem? Maybe there is a way to force cmake to use find module in `~/vcpkg/installed/x64-linux/share/xalan-c`?

--- a/ports/xalan-c/CONTROL
+++ b/ports/xalan-c/CONTROL
@@ -1,5 +1,5 @@
 Source: xalan-c
-Version: 1.11-7
+Version: 1.11-8
 Homepage: https://github.com/apache/xalan-c
 Description: Xalan is an XSLT processor for transforming XML documents into HTML, text, or other XML document types
 Build-Depends: xerces-c

--- a/ports/xalan-c/portfile.cmake
+++ b/ports/xalan-c/portfile.cmake
@@ -7,7 +7,7 @@ vcpkg_from_github(
     SHA512 e0f095b7031394c39c8e0fdca1f820c4222466f8c6e9df7bc40a21f9ca0e9291b7b6cdfb0a2d67db275ae97d7a7cdd447637102639e74716f0fb23a946b30ebe
     PATCHES
         fix-win-deprecated-err.patch
-		fix-missing-dll-error.patch
+        fix-missing-dll-error.patch
 )
 
 vcpkg_configure_cmake(
@@ -18,9 +18,9 @@ vcpkg_configure_cmake(
 vcpkg_install_cmake()
 
 if(EXISTS ${CURRENT_PACKAGES_DIR}/cmake)
-    vcpkg_fixup_cmake_targets(CONFIG_PATH cmake TARGET_PATH share/xalan-c)
+    vcpkg_fixup_cmake_targets(CONFIG_PATH cmake TARGET_PATH share/xalanc)
 else()
-    vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/XalanC TARGET_PATH share/xalan-c)
+    vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/XalanC TARGET_PATH share/xalanc)
 endif()
 
 # cleanup

